app/views/events/index.html.erb

```diff
@@ -5,25 +5,101 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Título de tu Página</title>
     <!-- Enlaces a tus hojas de estilo (CSS) o scripts (JavaScript) -->
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
 </head>
 <body>
 
 <%= render 'navbar' %>
 
-<div class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-1">
-  <% @event.each do |event| %>
-    <div class="col">
-      <div class="card h-100">
-        <div class="card-body">
-          <h3 class="card-title"><%= event.titulo %></h3>
-          <p class="card-text"><strong>Descripción:</strong> <%= event.descripcion %></p>
-          <p class="card-text"><strong>Fecha:</strong> <%= event.fecha %></p>
-          <p class="card-text"><strong>Ubicacion:</strong> <%= event.ubicacion %></p>
-          <p class="card-text"><strong>Costo:</strong> <%= event.costo %></p>
-        </div>
+<!-- Button trigger modal -->
+<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+  Añadir Evento
+</button>
+
+<!-- Modal -->
+<div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
Also you can make your code more accesible with this **[html tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)**
But I dont think its a good idea to do forms in a modal but thats my opinion